Fitness/Controllers/HomeController.swift

```diff
@@ -34,6 +35,7 @@ class HomeController: UIViewController {
     var tags: [Tag] = []
     var didSetupHeaderShadow = false
     var didRegisterCategoryCell = false
+    var pros = ProBio.getAllPros()
```

Comment:
we should be using `private var` if they're only being used in this class, here and everywhere else.